functions/Invoke-DbaDiagnosticQuery.ps1

```diff
@@ -189,7 +244,12 @@ Then it will export the results to Export-DbaDiagnosticQuery.
             }
 
             if (!$instanceOnly) {
-                $databases = $server.Query("Select Name from sys.databases where name not in ('master', 'model', 'msdb', 'tempdb')")
+                if (-not $DatabaseName) {
```

Comment:
changing the parameter at the top, remember to change this too

Reply:
Update this to the `$Database` parameter